### ISSUE TITLE
(FloatPanel) Error rendering Bokeh items: DOMException: Failed to execute 'drawImage' on 'CanvasRenderingContext2D'

### ISSUE THREAD

**Issue by ivandorte** [holoviz/panel#4855]
panel==1.0.0rc9
bokeh>=3.1.1,<3.2.0
holoviews==1.16.0a9

OS: Ubuntu 22.04.2 LTS, WSL 2
Tested in Chrome and Edge

When I try to render a FloatPanel with holoviews==1.16.0a9 library installed, I get the following error message in both Chrome and Edge consoles:

```log
Error rendering Bokeh items: DOMException: Failed to execute 'drawImage' on 'CanvasRenderingContext2D': The image argument is a canvas element with a width or height of 0.
    at g.blit_webgl (http://localhost:5006/static/js/bokeh.min.js?v=c8d721b75531cff043feef626ae0d6e29a26031fae26bba93c1baa962ecfb1b43f211027df3a0cf7772f30afed8c38198d70677b8967e581ca7e605358c68465:464:2802)
    at _._paint_levels (http://localhost:5006/static/js/bokeh.min.js?v=c8d721b75531cff043feef626ae0d6e29a26031fae26bba93c1baa962ecfb1b43f211027df3a0cf7772f30afed8c38198d70677b8967e581ca7e605358c68465:582:15891)
    at _._actual_paint (http://localhost:5006/static/js/bokeh.min.js?v=c8d721b75531cff043feef626ae0d6e29a26031fae26bba93c1baa962ecfb1b43f211027df3a0cf7772f30afed8c38198d70677b8967e581ca7e605358c68465:582:15197)
    at _.paint (http://localhost:5006/static/js/bokeh.min.js?v=c8d721b75531cff043feef626ae0d6e29a26031fae26bba93c1baa962ecfb1b43f211027df3a0cf7772f30afed8c38198d70677b8967e581ca7e605358c68465:582:14153)
    at _._after_layout (http://localhost:5006/static/js/bokeh.min.js?v=c8d721b75531cff043feef626ae0d6e29a26031fae26bba93c1baa962ecfb1b43f211027df3a0cf7772f30afed8c38198d70677b8967e581ca7e605358c68465:582:13990)
    at _.after_layout (http://localhost:5006/static/js/bokeh.min.js?v=c8d721b75531cff043feef626ae0d6e29a26031fae26bba93c1baa962ecfb1b43f211027df3a0cf7772f30afed8c38198d70677b8967e581ca7e605358c68465:547:6160)
    at _.compute_layout (http://localhost:5006/static/js/bokeh.min.js?v=c8d721b75531cff043feef626ae0d6e29a26031fae26bba93c1baa962ecfb1b43f211027df3a0cf7772f30afed8c38198d70677b8967e581ca7e605358c68465:547:5540)
    at _.invalidate_layout (http://localhost:5006/static/js/bokeh.min.js?v=c8d721b75531cff043feef626ae0d6e29a26031fae26bba93c1baa962ecfb1b43f211027df3a0cf7772f30afed8c38198d70677b8967e581ca7e605358c68465:547:6867)
    at _.after_render (http://localhost:5006/static/js/bokeh.min.js?v=c8d721b75531cff043feef626ae0d6e29a26031fae26bba93c1baa962ecfb1b43f211027df3a0cf7772f30afed8c38198d70677b8967e581ca7e605358c68465:547:6560)
    at _.r_after_render (http://localhost:5006/static/js/bokeh.min.js?v=c8d721b75531cff043feef626ae0d6e29a26031fae26bba93c1baa962ecfb1b43f211027df3a0cf7772f30afed8c38198d70677b8967e581ca7e605358c68465:547:6487)
```

And the FloatPanel does not render:

![floatpanel_error](https://github.com/holoviz/panel/assets/1726395/666ba170-e1e9-4e33-8a2c-717ecc2f5995)

This not happens with holoviews==1.16.0a2, in fact FloatPanel works fine.

Code to reproduce the issue:

```python
import holoviews as hv
import numpy as np
import panel as pn

pn.extension("floatpanel")

# Generate some random data
data = np.random.normal(size=1000)

# Calculate the histogram
frequencies, edges = np.histogram(data, bins=50)

# Plot the histogram
hist_plot = hv.Histogram((edges, frequencies))

# Embed the histogram in a FloatPanel
floatpanel = pn.layout.FloatPanel(hist_plot, margin=20)

# Create the dashboard and turn into a deployable application
pn.template.FastListTemplate(
    main=[pn.Column(floatpanel, width=400, height=400)]
).servable()
```

**Comment by philippjfr** [comment]
Seems like an upstream issue in Bokeh or HoloViews, will close.

**Comment by ianthomas23** [comment]
This is fixed in Bokeh by bokeh/bokeh#13140 which will be in the upcoming 3.2 release.
